app/views/tasks/index.html.erb

```diff
@@ -3,7 +3,22 @@
 <hr />
 <div class="list">
   <% @tasks.each do |task| %>
-    <div class="task"><%= link_to task.name, "/tasks/#{task.id}" %>
-    <%= button_to "Delete", { action: :destroy, id: task.id }, :class => "button", method: :delete, data: { confirm: 'Are you sure?' } %></div>
+    <% if task.completed_at.nil? %>
```

Comment:
A lot of this code is the same for both the `if` and the `else` cases. How might we DRY this out a bit to avoid repeating parts like `<div class="task">` ?